### DISCUSSION TITLE
exclude package 与 process name 有何差异

### DISCUSSION THREAD

**Discussion by NiuHK** [MetaCubeX/mihomo#1639] [Q&A] [answered]
安卓上的exclude package 与 process name 有何差异？在执行筛选效率上哪个好啊

**Comment by NiuHK** [comment]
就是在纯tun情况下，二者的筛选逻辑以及效率有何区别呢

**Reply by mengdegege** [reply, marked as answer]
package 是tun上的配置，针对Android匹配包名，process name是规则上的匹配，不在同一个环节
